pytest.ini

```diff
@@ -0,0 +1,13 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
```

Comment:
I suggest removing that: addopts = -ra -q 
so we would know what tests are executed.